package config;

message RouterInfo {
	optional string listen_ip = 1;
	optional int32 listen_port = 2;
	optional int32 number = 3;
}

message RouterConfig {
	repeated RouterInfo router_list = 1;
}